src/components/index.ts

```diff
@@ -17,10 +17,13 @@ export * from "./table";
 export * from "./xDragBoxLayer";
 export * from "./yDragBoxLayer";
 
-export class Alignment {
-  static TOP = "top";
-  static BOTTOM = "bottom";
-  static LEFT = "left";
-  static RIGHT = "right";
-  static CENTER = "center";
-}
+/**
+ * @deprecated just use string literals for alignment
```

Comment:
I'm not sure I follow - classes already do this:

```ts
class Point { ... }

function makePoint(): Point { // here we're referring to the Point type
  return new Point(); // here we're referring to the Point variable
}
```

We say "the Point constructor creates an object of type Point" -- same thing here. "BarOrientation.vertical is a member of type BarOrientation"